domains/joyal-jijo.json

```diff
@@ -0,0 +1,14 @@
+{
+    "description": "joyal portfolio",
+    "repo": "https://github.com/joyal-jij0/portfolio",
+    "owner": {
+        "username": "joyal-jij0",
+        "email": "",
+        "discord": "joyal_jij0"
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```

Comment:
Error 404

Reply:
Hey, can't really understand where is the problem. So can you help me with that ?
Here's my website : 
https://joyal-jijo.netlify.app